vmhost/vmhooks/baseOps.go

```diff
@@ -1452,6 +1453,144 @@ func CreateAsyncCallWithTypedArgs(host vmhost.VMHost,
 	return 0
 }
 
+// CreateAsyncV3Call VMHooks implementation.
+// @autogenerate(VMHooks)
+func (context *VMHooksImpl) CreateAsyncV3Call(
+	destOffset executor.MemPtr,
```

Comment:
everything should be managed. not PTR. Or have the same call with managed buggers as well.